packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -298,6 +298,44 @@ skipInNode(describe)('plugin-meetings', () => {
         assert.exists(alice.meeting.members.selfId, 'selfId not present');
       });
 
+      it('alice resolution is 1080p, bob receives 1080p', () => {
+        testUtils.delayedPromise(alice.meeting.mediaProperties.setLocalQualityLevel('1080p'));
+
+        testUtils.waitForEvents([
+          {scope: alice.meeting, event: 'media:ready'}
```

Comment:
this check should verify also the event payload's type is "local" and the stream contains a track with correct settings and we should use a `match` argument for `waitForEvents()` to do these checks, otherwise this test can be flaky as it will fail if any other event is sent first.

Reply:
Actually we don't need to wait for this event as this gets triggered when we call `addMedia()`.